feature-tests/utils/invoke-apigateway-lambda.ts

```diff
@@ -0,0 +1,13 @@
+import { InvokeCommand, LambdaClient } from '@aws-sdk/client-lambda';
+
+export async function invokeApiGateWayLambda(testUserId: string, historyValue: any) {
```

Comment:
```suggestion
export async function invokeApiGateWayLambda(testUserId: string, historyValue: boolean) {
```
Sounds like you might be removing this function anyway, but `historyValue` should be a boolean type.

Reply:
updated